src/solver/misc/options.cpp

```diff
@@ -295,6 +295,15 @@ void checkAndCorrectSettingsAndOptions(Settings& settings, Data::StudyLoadOption
     }
 }
 
+static bool containsOnlyASCIIcharsOnWindows(const std::string& path)
+{
+#if defined(_WIN32) || defined(_WIN64)
```

Comment:
```suggestion
#if defined(_WIN32)
```
> _WIN32 Defined as 1 when the compilation target is 32-bit ARM, 64-bit ARM, x86, x64, or ARM64EC. Otherwise, undefined.

https://learn.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170

Reply:
Done